VTEX - Ad Network API.json

```diff
@@ -634,6 +1036,42 @@
                     "example": "color/blue"
                 }
             },
+            "sponsoredCount": {
+                "in": "query",
+                "name": "sponsoredCount",
+                "description": "Amount of sponsored products to be returned.",
```

Comment:
Maybe it would be worth it to say who it would be returned to, for a better understanding.